src/command/helpModule.ts

```diff
@@ -40,6 +41,34 @@ export default class HelpModule extends Module {
             );
         }, initialEmbed);
 
-        await msg.channel.send(embed);
+        const fallbackContent = modules.reduce((message, module) => {
```

Comment:
`message` is too similar to `msg`, might be confusing for future readers.

Reply:
This is on line 44 in the reducer?